src/webview_provider/datapilotPanel.ts

```diff
@@ -43,13 +45,10 @@ export class DataPilotPanel extends AltimateWebviewProvider {
       dbtTerminal,
     );
 
-    commands.registerCommand(
-      "dbtPowerUser.resetDatapilot",
-      () =>
-        this._panel?.webview.postMessage({
-          command: "datapilot:reset",
-          args: {},
-        }),
+    commands.registerCommand("dbtPowerUser.resetDatapilot", () =>
+      this.sendResponseToWebview({
```

Comment:
used a common method `sendResponseToWebview` instead of `this.panel.webview.postMessage`